ci-operator/config/openshift-cloud-team/rebasebot/openshift-cloud-team-rebasebot-main.yaml

```diff
@@ -47,6 +47,58 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: cloud-provider-ibm
+  cron: 10 12 * * Mon,Thu
+  steps:
+    test:
+    - as: cloud-provider-ibm
+      commands: |
+        rebasebot --source https://github.com/IBM-Cloud/cloud-provider-ibm:release-1.22 \
```

Comment:
Maybe get branch name from env variables? I expect that it will change for next release cycle.